.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
Would it make sense to automatically assign the 'bug' label on these issues? Or should this happen after triage?

Reply:
Hmm, we would probably want a separate "feature request" template in that case (not all features add API)

Reply:
Perhaps I should have a "blank template" as well, for such things.